### DISCUSSION TITLE
thanks

### DISCUSSION THREAD

**Discussion by southguy** [TENK-DAO/tenk#99] [Ideas]
thanks

**Comment by willemneal** [comment]
> if you dont make detailed page you guys literally killing near #eco system..

🤷 Not sure what you mean.  Most big nft collection projects use this so not sure how we are killing the ecosystem.  I agree it could be better documented, but we have also been working on tools, used in this project, that generate the documentation for the contract and even embed the contract API in the binary allowing you to get an autogenerated website for your contract, see https://raen.dev/.  I have been spending more of my time in these efforts because I believe that will have a bigger impact on the ecosystem.

I am flattered that you think this project's success is linked with the success of the near ecosystem, but we are human and while we provide everything, we have yet to have much community contribution, which is to be expected when other projects are trying to make a name for themselves.

**Comment by starpause** [comment]
> detailed knowledge page about how make nft contract.

Maybe you are looking for https://github.com/near-examples/nft-tutorial
